### DISCUSSION TITLE
Next Version of OSDM - Backup Compatible or Not?

### DISCUSSION THREAD

**Discussion by schlpbch** [UnionInternationalCheminsdeFer/OSDM#578] [Polls]
Over the years, we have accumulated a list of deprecated attributes as well as some redundant flows in the OSDM v.3.3 version. 

The question discuss it whether we should to address these topics with the next release. This would imply breaking backward compatibility for some features,

**Comment by ralfbayer-db** [comment]
I need a couple of days to discuss internally. Regard, Ralf

**Reply by ralfbayer-db** [reply]
We discussed internally at DB, and we prefer cleaning up.
